plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -199,15 +199,26 @@ private static void generateEnumAttributeSchema(JSONObject attributeSchemaTempla
     }
 
     public static void rootConfigGeneration() throws Exception {
+
+
         DefaultConfiguratorRegistry registry = new DefaultConfiguratorRegistry();
         final ConfigurationContext context = new ConfigurationContext(registry);
+        context.setMode("JSONSchema");
         for (RootElementConfigurator root : RootElementConfigurator.all()) {
             final CNode config = root.describeStructure(root.getTargetComponent(context), context);
             final Mapping mapping = config.asMapping();
             final List<Map.Entry<String, CNode>> entries = new ArrayList<>(mapping.entrySet());
             for (Map.Entry<String, CNode> entry : entries) {
                 System.out.println(entry.getKey() + " " + entry.getValue().toString());
```

Comment:
@casz `io.jenkins.plugins.casc.ConfiguratorException: Can't read attribute 'adminAddress' from class jenkins.model.JenkinsLocationConfiguration`
Quite a few of the attributes return this exception. Should we just ignore them for now ?

Reply:
No, we should try and access the getter? [`getAdminAddress`](https://javadoc.jenkins.io/jenkins/model/JenkinsLocationConfiguration.html#getAdminAddress--)

Reply:
@casz It is for every single attribute `io.jenkins.plugins.casc.ConfiguratorException: Can't read attribute 'authenticators' from class jenkins.security.QueueItemAuthenticatorConfiguration`

Reply:
@casz We still cannot iterate over the values though :sweat: